### PR TITLE
Update README in anticipation of #377

### PR DESCRIPTION
**Pull request by hongaar**

### CODE REVIEW

**Comment by flibustier** [comment]
St. hongaar, still ahead! 😄

**Comment by PlasmaPower** [comment]
I was thinking about this, maybe "democracy + meritocracy"? Democracy is still the most important element IMO.

**Comment by PlasmaPower** [comment]
Oh and you might want to change "democracy" to "direct democracy" in case we get another type of democracy later.

File: README.md [hunk 1]
```diff
@@ -57,10 +57,10 @@ Votes on a PR are determined through following mechanism:
   depending on the local server time.
 * While the voting process is going, users can change their reactions and edit
   their comments, effectively changing their vote.
-
```
Comment (flibustier) [inline README.md:60]: We see the influence of PlasmaPower here!

**Comment by hongaar** [comment]
@PlasmaPower 👍

**Review by andrewda — APPROVED** [review]

**Review by phil-r — APPROVED** [review]

**Pull request merged**